swagger.yaml

```diff
@@ -2221,3 +2227,15 @@ definitions:
         format: ipv4
         description: The IPv4 address that was allocated by DHCP for Ingress VIP.
 
+  cluster-validation-id:
```

Comment:
I don't see where this is used in swagger.
If it is not used in swagger, why define it here and not in the code?

Reply:
I did it similar to host-validation-id that also defined in the swagger. I think it's important for the UI. @ori-amizur ?